sdk/storage/azure-storage-file-datalake/tests/test_directory_async.py

```diff
@@ -122,9 +108,10 @@ async def test_create_directory_async(self, datalake_storage_account_name, datal
         created = await directory_client.create_directory(content_settings=content_settings)
 
         # Assert
-        self.assertTrue(created)
+        assert created
 
     @DataLakePreparer()
+    @recorded_by_proxy_async
     async def test_create_directory_owner_group_acl_async(self, datalake_storage_account_name,
```

Comment:
Another instance of the multi-line function prototype breaking my script. Would you mind updating this to use kwargs?

Looks like this happens more times in this file.

Reply:
Good catch, I will go back and fix all these instances!